### ISSUE TITLE
Unlabeled is prepended before labelled radio elements while in elements list

### ISSUE THREAD

**Issue by skorasaurus** [nvaccess/nvda#11165]
nvda prepends the world "unlabelled" before radio elements that are labelled when a user is browsing through form fields through the elements list (insert + f7)

### Steps to reproduce:

1. Open following url in chrome https://scottaohara.github.io/a11y_styled_form_controls/src/radio-button
2. Insert + F7 to open elements list, change to form fields, 
3. Listen and hear "unlabeled radio button checked option 1" before each radio input option
 
### Expected behavior:

1. Open page; https://scottaohara.github.io/a11y_styled_form_controls/src/radio-button
2. Insert + F7 to open elements list, change to form fields
3. Listen to "option 1, radio button checked"  (This is the behavior on firefox 76). In Firefox, the 

This behavior (of prepending "unlabeled" does not occur when browsing through elements by tab. What occurs when the tab focus is on the first radio element is:
```
Visually Hidden Radio Buttons  grouping
Option 1  radio button  checked  1 of 3
```

#### NVDA installed/portable/running from source:
NVDA portable,
#### NVDA version:
 2020.01; no modifications to settings made after first opening; 

#### Windows version:
windows 10; 

#### Name and version of other software in use when reproducing the issue:
crhome 81.0.4044.138

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Can reproduce this on NVDA 2019.x; 

Can confirm that NVDA 2018.1 is the most recent version with Chrome where this is not reproducable.

cc @WestonThayer and @scottaohara for actually doing most of the lifting on the report.

**Comment by Adriani90** [comment]
This seems only reproducible in Chromium browsers.

cc: @aleventhal

**Comment by Bumadharshiny** [comment]
This issue is happening for Tablist / tabs as well

![image](https://user-images.githubusercontent.com/88369651/128214971-01874327-2f70-4ad4-bba5-8bcf89927958.png)

**Comment by scottaohara** [comment]
This issue continues to occur with Chrome 92 and NVDA 2021.1.

This issue *does not occur* with Edge 92 and NVDA 2021.1.

**Comment by Bumadharshiny** [comment]
For tablist/tabs - issue is occuring in Edge 92 (NVDA 2021.1) as well

**Comment by WestonThayer** [comment]
@scottaohara I'm not experiencing the difference between Chrome 92.0.4515.131. and Edge 92.0.902.62, the issue reproduces in both. I'd expect this because in terms of OS a11y APIs, they should be identical 99% of the time. Are your versions the same?

**Comment by scottaohara** [comment]
Same versions, yes.  Re-verified again just now.  Edge does not show "unlabeled" in front of radio button names in dialog.  Chrome does.

**Comment by WestonThayer** [comment]
Bizarre. Windows version? I'm on 21H1, Professional edition. Will try on a few other PCs when I get a chance.

**Comment by skorasaurus** [comment]
for me:
Windows 10 Enterprise
Version	21H1
Installed on	‎2/‎20/‎2021
OS build	19043.1110
Experience	Windows Feature Experience Pack 120.2212.3530.0

NVDA 2021.1; Chrome  92.0.4515.131 and Edge 92.0.902.67; I'm able to reproduce the issue with both browsers.

**Comment by mfairchild365** [comment]
This is also happening with checkboxes. Tested against this page: https://a11ysupport.io/tests/html/html/checkboxes.html

Tested with Windows 11 21H2 and NVDA 2022.2.2. The issue occurs in Chrome 105 and Edge 105, but not Firefox 104.

These issues are not present in JAWS 2022 with the above combinations.

**Comment by Adriani90** [comment]
This is covered in this Chromium bug:
https://bugs.chromium.org/p/chromium/issues/detail?id=1433153

**Comment by Adriani90** [comment]
@maniktr, @Bumadharshiny can you please upload a minimal test case with tablists / tabs where this issue is reproducible? Here are a lot of blind developers and we need an alternative to the screenshots for testing. Otherwise it is quite impossilbe to investigate further. Thanks for your understanding.

**Comment by Adriani90** [comment]
and please feel free to also comment on the Chromium bug I have referenced above so it gains some attention from the Chrome developers again. Thanks.

**Comment by maniktr** [comment]
> @maniktr, @Bumadharshiny  can you please upload a minimal test case with tablists / tabs where this issue is reproducible? Here are a lot of blind developers and we need an alternative to the screenshots for testing. Otherwise it is quite impossilbe to investigate further. Thanks for your understanding.

@Adriani90 Adding the Test case, Actual and Expected Behaviors for the reference:

## NVDAprepends the world "Unlabelled" before labelled Tab elements that when a user is browsing through form fields through the elements list (insert + f7)

#### Steps to reproduce

1. Open the URL in Chrome Browser: [https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/#ex_label](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/#ex_label) 
2. Insert + F7 to open Elements List of NVDA application
3. Select the Form fields radio option from the options
4. As you can see from the below content, the labelled tabs are being shown as "Unlabeled" and refer the screenshot

### Actual Behavior:
```
Unlabeled; tab; selected; Maria Ahlefeldt
Unlabeled; tab; Carl Andersen
Unlabeled; tab; Ida da Fonseca
Unlabeled; tab; Peter Müller
```
![image](https://github.com/nvaccess/nvda/assets/54179547/fe27fefd-bae4-4327-8701-cc96007c0bb1)

### Expected Behavior:

1. Open the URL in Edge Browser: [https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/#ex_label](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/#ex_label) 
2. Insert + F7 to open Elements List of NVDA application
3. Select the Form fields radio option from the options
4. As you can see from the below content, Tabs are labelled correctly and refer the screenshot

```
Maria Ahlefeldt; tab; selected; Maria Ahlefeldt
Carl Andersen; tab; Carl Andersen
Ida da Fonseca; tab; Ida da Fonseca
Peter Müller; tab; Peter Müller
```
![image](https://github.com/nvaccess/nvda/assets/54179547/7fb16e65-e8eb-4319-961f-8d374be0de3b)

#### Note: Tab labels are showing properly on Edge but not on Chrome.

#### System configuration
NonVisual Desktop Access

**NVDA version:**
2023.3

**Windows version:**
windows 10;

**Name and version of other software in use when reproducing the issue:**
 Chrome 120.0.6099.71 and Edge 119.0.2151.97.

**Other questions**
Does the issue still occur after restarting your computer?
yes

**Have you tried any other versions of NVDA? If so, please report their behaviors.** 
NO

**Please look into this issue and let me know if this will be fixed in the future ?**

**Comment by JMac118** [comment]
This issue still persists with "Unlabeled tabs" in NVDA's Elements List in the "Form fields" section across these browsers:

**Versions Tested:**
- NVDA: 2025.3.2 (2025.3.2.53588)
- Firefox: 146.0.1 (64-bit)
- Edge: 143.0.3650.139 (Official build) (64-bit)
- Chrome: 143.0.7499.193 (Official Build) (64-bit)

**Tested with:**
- W3C ARIA APG tab example: [https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/)
- Custom test implementations with `aria-label`, `aria-labelledby` and plain text content

**Behavior:**
- Elements List displays: "Unlabeled; tab; selected; [tab name]"
- Normal navigation (Tab/Arrow keys) correctly announces tab names
- Issue affects tabs with `aria-label`, `aria-labelledby`, and plain text content

Confirming @a1ms1's October 2024 report that Firefox is now also affected.

**Comment by joanlambert** [comment]
This bug was opened 6 years ago and still exists. What do we need to do to get it fixed?
